apps/store/src/features/widget/ProductItem.tsx

```diff
@@ -144,9 +144,11 @@ export const ProductItem = (props: Props) => {
   )
 }
 
-export const ActionButton = forwardRef<HTMLButtonElement, ButtonProps>((props, ref) => {
-  return <Button ref={ref} size="medium" variant="secondary-alt" {...props} />
-})
+export const ActionButton = forwardRef<HTMLButtonElement, ButtonProps<React.ElementType>>(
```

Comment:
Should we say those `ActionButton`s are suppose to be used as `button`s? I mean, I understand the flexibility now. I'm just wondering if at this point we already decide the shape of it.

```suggestion
export const ActionButton = forwardRef<HTMLButtonElement, ButtonProps<'button'>>(
```

Reply:
Sure! Sorry I merged before noticing this comment.